test/testHelpers.ts

```diff
@@ -9,11 +9,11 @@ import { MobileWalletExport } from '../src/wallet/types';
  * Creates a client to communicate with a local concordium-node
  * used for automatic tests.
  */
-export function getNodeClient(): ConcordiumNodeClient {
+export function getNodeClient(address = '127.0.0.1'): ConcordiumNodeClient {
```

Comment:
You changed this to be a parameter, but is it ever used?

Reply:
Was to make it easier to target a specific node (e.g. stagenet) for some tests, without messing up the rest.